test/runpytests.jl

```diff
@@ -31,46 +8,26 @@ import Main.AwkwardPyCall: julia_array_to_python
         AwkwardArray.PrimitiveArray([1.1, 2.2, 3.3, 4.4, 5.5]),
     )
     # Test case 1: Check if the function returns an awkward array
-    @test isa(julia_array_to_python(array), PyObject)
+    @test isa(julia_array_to_python(array), Py)
 
     # Test case 2: Check if the awkward array has the correct layout
     py_array = julia_array_to_python(array)
-    @test typeof(py_array) == PyObject
-    @test pyimport("awkward").to_list(py_array) == [[1.1, 2.2, 3.3], [], [4.4, 5.5]]
+    @test typeof(py_array) == Py
+    ak_array = pyconvert(Vector, pyimport("awkward").to_list(py_array))
+    @test ak_array == [[1.1, 2.2, 3.3], [], [4.4, 5.5]]
 
 
 end
 
-import Main.AwkwardPyCall: python_array_to_julia
+using AwkwardArray: python_array_to_julia
 
 # Test python_array_to_julia function
 @testset "python_array_to_julia tests" begin
     py_array = pyimport("awkward").Array([[1.1, 2.2, 3.3], [], [4.4, 5.5]])
 
+    array = python_array_to_julia(py_array)
     # Test case 1: Check if the function returns an awkward array
-    @test isa(
-        python_array_to_julia(py_array),
-        AwkwardArray.ListOffsetArray{
-            SubArray{Int64,1,Vector{Int64},Tuple{UnitRange{Int64}},true},
-            AwkwardArray.PrimitiveArray{
-                Float64,
-                SubArray{Float64,1,Vector{Float64},Tuple{UnitRange{Int64}},true},
-                :default,
-            },
-            :default,
-        },
-    )
+    @test array isa AwkwardArray.ListOffsetArray
 
-    # Test case 2: Check if the awkward array has the correct layout
-    array = python_array_to_julia(py_array)
-    @test typeof(array) == AwkwardArray.ListOffsetArray{
-        SubArray{Int64,1,Vector{Int64},Tuple{UnitRange{Int64}},true},
-        AwkwardArray.PrimitiveArray{
-            Float64,
-            SubArray{Float64,1,Vector{Float64},Tuple{UnitRange{Int64}},true},
-            :default,
-        },
-        :default,
```

Comment:
there must be a better way to check layout other than hard coding all these types, these type parameters shouldn't be considered fully stable